update articles link between assets & products (#431)
akeneo/pim-helpcenter: content/md/first-steps/manage-your-attributes.md

1. Select the attribute type you want:
  ![image](../img/Settings_Attributes_Create2.png)
1. The attribute edit form is displayed, fill in the following properties:  
    Under `General Parameters`:
    - A **code** (required) to identify the attribute.
    - A **type** based on the attribute type selected in the previous screen.

>     - An **attribute group** (required) in which the new attribute will be added to.    

Reviewer: ```suggestion
    - An **attribute group** (required) to which the new attribute will be added.    
```

    - **Read only** (Yes/No) **_(Enterprise Edition only)_**: to prevent this attribute from being edited through the UI, it will only be updated through imports, API or rules

>     - **Usable in grid** (Yes/No): if you want to filter on this attribute in the product's grids or use it as a column.

Reviewer: ```suggestion
    - **Usable in grid** (Yes/No): if you want to filter on this attribute in the product grid or use it as a column.
```

Reviewer: in the blue non-numbered above line 32, the last word should be "properties"

Reviewer: in the blue non-numbered line above line 79, I suggest "Each attribute has specific properties to add (or "to provide") more control over its value" or "...over attribute values."

- **Reference entity single link attribute** & **reference entity multiple links attribute _(Enterprise Edition only)_**:
  - *Related reference entity*: the reference entity linked to this attribute
- **Reference data simple select attribute** & **reference data multi select**:

>   - *Reference data type*: choose an existing reference data to link to your attribute

Reviewer: ```suggestion
  - *Reference data type*: choose an existing reference entity to link to your attribute
```

Reviewer: A minor point, but if "a reference data" is the right word to use instead of "entity", it needs another word at the end, such as "a reference data value"

Reviewer 2: Yes, it's reference data, the CE feature.

Reviewer 3: ```suggestion
  - *Reference data type*: choose an existing reference data value to link to your attribute
```
